gtsfm/utils/graph.py

```diff
@@ -0,0 +1,30 @@
+"""Utilities for performing graph operations.
```

Comment:
Can we be more specific here?

Technically GTSFM is one big graph. So it's not clear what kind of graph this is.

Can we name the file something like `gtsfm/utils/connected_components.py`?

Reply:
I wanted this file to be the place where we can put all graph (i.e. operations on nodes and edges) related operations. We currently just need connected_components, but in the future we might need some more.

For example, we might have a system to break the global graph into smaller densely connected components and join them using re-sectioning? Or figuring out which part of graph needs more edges for averaging and figuring out how to recover two view estimated on required edges better.

Reply:
ok. well i dont feel strongly about it. we can leave it then